client/src/map.js

```diff
@@ -24,13 +25,49 @@ function Map(props) {
   const [currentCenter, setCurrentCenter] = useState(defaultCenter);
   const [selectedMarker, setSelectedMarker] = useState(null);
 
+  useEffect(() => {
+    setRealtimeData(props.realtimeData);
+  }, [props.realtimeData]);
+
   const mapRef = React.useRef();
   const onMapLoad = React.useCallback((map) => {
     mapRef.current = map;
   }, []);
 
+  // called when the maps bounds are changed e.g. when a user drags the map
+  const onBoundsChanged = () => {
+    // TODO: uncomment this code once the server supports 'geo-position-update'
+    /*
+    let lat = mapRef.current.getCenter().lat();
+    let lng = mapRef.current.getCenter().lng();
+    let radius = getBoundingSphereRadius();
+
+    let message = {
+      "type": "geo-position-update",
+      "payload": {
+        "radius": radius,
+        "position": {
+          "latitude": lat,
+          "longitude": lng
+        }
+      }
+    };
+
+    props.wsSend(JSON.stringify(message);
+    */

```

Comment:
We are in the process of changing the format of the `geo-position-update` message with regards to some comments from @Caw3 in #17. I suggest we wait with merging this PR until we've discussed this on our meeting on Monday.